definitions/npm/lodash_v4.x.x/flow_v0.153.x-v0.154.x/lodash_v4.x.x.js

```diff
@@ -591,7 +591,7 @@ declare module "lodash" {
     filter:
       & (<T>(array?: ?$ReadOnlyArray<T>, predicate?: ?Predicate<T>) => Array<T>)
       & (<A, T: ReadOnlyIndexerObject<A>>(
-        object: T,
+        object?: ?T,
```

Comment:
Si t's a bit wider thant removing dissoc?

Reply:
Yeah so lodash never had all it's tests ever passing so I took the opportunity to do some cleanup work while I was in the area. `filter()` and `filter(null)` both return empty array.

Should I just split them into separate PR's though?